fix:

rav3n11 opened the pull request:
- ignored lines in backend/src/server/httpUtils.test.ts that sonarcloud mistakes for mongodb credentials ( they were mocks)
- ignored all *.json files from being included in sonarcloud coverage metrics.

jacob-js commented on sonar-project.properties:
```diff
@@ -18,7 +18,7 @@ sonar.coverage.exclusions=\
   backend/test/**, backend/src/_test_utilities/**, backend/src/**/_test_utilities/**, backend/src/**/_test_data_/**, backend/src/**/*.test.ts, backend/*.js, \
   frontend/test/**, frontend/src/_test_utilities/**, frontend/src/**/*.test.ts, frontend/src/**/*.stories.ts, frontend/src/**/*.stories.tsx, frontend/src/**/*.test.tsx, frontend/*.js, \
-  api-specifications/src/**/*.test.ts, api-specifications/src/_test_utilities/**, api-specifications/*.js
+  api-specifications/src/**/*.test.ts, api-specifications/src/_test_utilities/**, api-specifications/*.js, *.json
```
I'd recommend putting the `*.json` on its own line. If you pay attention to the lines above, each line represents a folder containing the files to be excluded. Therefore, the new line you're adding is not within either folder listed there. It's better to place it at the top of all folders.

rav3n11 commented:
Sonarcloud check is failing because the new configuration I've added will not be complete unless sonarcloud get's the current PR number for the PR analysis. Aparently it's a required field. I've modified the sonarcloud workflow file to run a bash script that gets the PR number from the ${GITHUB_REF} environment variable and modifies the properties file.

rav3n11 commented:
This may not be the best way to accomplish the desired outcome...
